--- a/scimschema/scimschema/_model/schema_response.py
+++ b/scimschema/scimschema/_model/schema_response.py
@@ -14,9 +14,9 @@ class ScimResponse(dict):
         ) = self._get_meta_schemas(
             core_schema_definitions, extension_schema_definitions
         )
-        if len(self._core_meta_schemas) != 1:
+        if len(self._core_meta_schemas) > 1:
             raise AssertionError(
-                "Response must specify exactly one core schema - {}".format(
+                "Response must specify up to one core schema - {}".format(
                     ", ".join([s.id for s in self._core_meta_schemas])
                 )
             )
